flax/optim/dynamic_scale.py

```diff
@@ -86,7 +86,7 @@ def loss_fn(p):
   def value_and_grad(self, fun: Callable[..., Any],
                      argnums: Union[int, Sequence[int]] = 0,
                      has_aux: bool = False,
-                     axis_name: str = None,
+                     axis_name: Optional[str] = None,
```

Comment:
This was changed in the typing PEP, and type-checkers now ask that these be explicitly marked optional

Reply:
Makes sense